es-ds-docs/pages/molecules/pagination.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div>
+        <h1>Pagination</h1>
+        <p class="pb-200">
+            Extended from <nuxt-link
+                href="https://v3.primevue.org/paginator/"
+                target="_blank">
+                PrimeVue paginator
+            </nuxt-link>
+        </p>
+
+        <div class="mb-500">
+            <ul
+                id="paginated-list">
+                <li
+                    v-for="item in items.slice(first, first + perPage)"
+                    :key="item">
+                    {{ item }}
+                </li>
+            </ul>
+            <es-pagination
+                v-model:first="first"
+                :total-rows="rows"
+                :per-page="perPage"
+                list-id="paginated-list"
+                align="center" />
+        </div>
+
```

Comment:
Could we provide the props documentation here? That's especially important if we do change the API in any way.

Reply:
Good idea! added